### PR TITLE
US 1583733 Add missing language IDs - 14

### PR DESCRIPTION
**Pull request by v-thpra**
US 1583733 - Fix CATS P1 issues by adding missing language IDs to code blocks.
- Used "output" for command tree examples.
- Used "sql" for examples of SQL statements, and "csharp" for .NET Framework APIs that consume them.

Contributes to #2192

### CODE REVIEW

File: docs/framework/data/adonet/optimistic-concurrency.md [hunk 1]
```diff
@@ -61,13 +61,13 @@ In a multiuser environment, there are two models for updating data in a database
   
  Another technique for testing for an optimistic concurrency violation is to verify that all the original column values in a row still match those found in the database. For example, consider the following query:  
   
-```  
+```csharp
```
Comment (Youssef1313) [inline docs/framework/data/adonet/optimistic-concurrency.md]: Should be sql ?

File: docs/framework/data/adonet/optimistic-concurrency.md [hunk 2]
```diff
@@ -61,13 +61,13 @@ In a multiuser environment, there are two models for updating data in a database
...
 SELECT Col1, Col2, Col3 FROM Table1  
 ```  
   
  To test for an optimistic concurrency violation when updating a row in **Table1**, you would issue the following UPDATE statement:  
   
-```  
+```csharp
```
Comment (Youssef1313) [inline docs/framework/data/adonet/optimistic-concurrency.md]: Should be sql ?

File: docs/framework/data/adonet/optimistic-concurrency.md [hunk 3]
```diff
@@ -82,7 +82,7 @@ WHERE Col1 = @OldCol1Value AND
   
  If a column at your data source allows nulls, you may need to extend your WHERE clause to check for a matching null reference in your local table and at the data source. For example, the following UPDATE statement verifies that a null reference in the local row still matches a null reference at the data source, or that the value in the local row still matches the value at the data source.  
   
-```  
+```csharp
 UPDATE Table1 Set Col1 = @NewVal1
```
Comment (Youssef1313) [inline docs/framework/data/adonet/optimistic-concurrency.md]: Should be sql ?

**Comment by v-thpra** [comment]
@Youssef1313 Thanks for catching those in your review! I think you are exactly right. They are examples of SQL statements and the C# example is at the end of the article.

**Comment by v-thpra** [comment]
@mairaw All changes and conflicts have been addressed. This should be ready to merge. Thanks!

File: docs/framework/data/adonet/sql/authentication-in-sql-server.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ SQL Server supports two authentication modes, Windows authentication mode and mi
   
  With Windows authentication, users are already logged onto Windows and do not have to log on separately to SQL Server. The following `SqlConnection.ConnectionString` specifies Windows authentication without requiring users to provide a user name or password.  
   
-```  
+```csharp  
 "Server=MSSQL1;Database=AdventureWorks;Integrated Security=true;
```
Comment (gewarren) [inline docs/framework/data/adonet/sql/authentication-in-sql-server.md]:
```suggestion
"Server=MSSQL1;Database=AdventureWorks;Integrated Security=true;"
```

Also I'm not convinced this should be "csharp" but I'm late to this party so maybe that's been decided already.

**Comment by v-thpra** [comment]
@gewarren Thanks for fixing this!  I used csharp, because It's a connection string that is being passed to a C# method. We used csharp for some other connection strings in this batch. I think that's the closest language ID - but we could leave it blank or change it if doesn't fit well enough.

**Review by BillWagner — APPROVED** [review]
Thanks for the other reviews.

I'm happy with the changes, and I'll :shipit:

**Pull request merged**